teachertool/src/utils/browserUtils.ts

```diff
@@ -0,0 +1,12 @@
+export const isLocal = () => {
```

Comment:
Personally, I don't see an issue with just putting these inside the existing index.ts file in utils rather than creating a separate file for them, but it's not something I feel strongly about.

Reply:
Sure, makes sense. I'll move the functions.